src/pages/blog/recent-featured-posts/index.js

```diff
@@ -0,0 +1,182 @@
+    DotIcon
+} from './_style'
+import Layout from 'components/layout/layout'
+import { Carousel,Tabs } from 'components/elements'
+import { localize, WithIntl } from 'components/localization'
+
+//Settings for carrousel
+const settings = {
+    options: {
+        axis: 'y',
+        align: 'start',
+        slidesToScroll: 3,
+    },
+    view_port: {
+        height: 'auto',
+    },
+    vertical_container: {
+        flexDirection: 'column',
+        height: '464px',
+        width: '384px'
+    },
+}
+
+const RecentFeaturedPosts= () => {
+    const recent = recent_article_data;
+    const featured = featured_article_data;
+    const articles = article_data;
+    const articles_2 = article_data_2;
+
+    return(
+    <Layout>
+        <StyledContainer m="20px auto 0" fd="column" ai="center">
+             <StyledTabs
+                    tab_list={['recent_posts', 'featured_posts']}
+                    route_from="recent_featured_posts"
+                    is_left_aligned
+                >
+                    <Tabs.Panel label={localize('Recent posts')}>
+                        <ArticleContentWrapper>
+                            <LeftContent>
+                                <RedirectLink to={recent.link}>
+                                    <MainArticle image={recent.image} >
+                                        <Description>
+                                            <TagParentWrapper>
+                                                <TagWrapper>
+                                                    <StyledCategories>
+                                                        {recent.category}
+                                                    </StyledCategories>
+                                                </TagWrapper>
+                                            </TagParentWrapper>
+                                                <ArticleTitle>
+                                                    {recent.title}
+                                                </ArticleTitle>
+                                                <ArticleSubtitle>
+                                                    {recent.description}
+                                                </ArticleSubtitle>
+                                        </Description>
+                                        <BottomDescription>
+                                            {recent.date} <ClockIcon src={Clock}/>  {recent.reading_time} mins read
+                                        </BottomDescription>
+                                    </MainArticle>
+                                </RedirectLink>
+                            </LeftContent>
+                            <RightContent>
+                                <Carousel {...settings}> {
+                                    articles.map((article,idx) => {
+                                        return (
+                                            <RedirectLink to={article.link} key={idx}>
```

Comment:
Try to avoid using the index as a key. Can refer to this for more reference. https://medium.com/@vraa/why-using-an-index-as-key-in-react-is-probably-a-bad-idea-7543de68b17c
Instead, you could use the `article.title`